front/components/layout/MainFooter.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <footer>
+    <v-row>
+      <v-col cols="12" sm="6" class="text-center text-sm-left">
+        <n-link to="/about">About</n-link>
+        <n-link to="/faq">FAQ</n-link>
+        <n-link to="/terms">Terms &amp; Conditions</n-link>
+        <n-link to="/privacy">Privacy</n-link>
+      </v-col>
+      <v-col cols="12" sm="6" class="text-center text-sm-right">
+        <span>Copyright Daily Idea © 2020</span>
```

Comment:
2020 better to be dynamic from current date so we don't need to update it next year

Reply:
It was not in the ticket originally, I'd suggest it should go to new ticket about layout refactoring that will include footer?

Reply:
ye, no prob sounds fine